src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -0,0 +1,640 @@
+        biodiversityAttemptFillChanceSpinBox = GetNode<SpinBox>(BiodiversityAttemptFillChancePath);
+        biodiversityFromNeighbourPatchChanceSpinBox = GetNode<SpinBox>(BiodiversityFromNeighbourPatchChancePath);
+        biodiversitySplitIsMutatedCheckBox = GetNode<CustomCheckBox>(BiodiversitySplitIsMutatedPath);
+        biodiversityNearbyPatchIsFreePopulationCheckBox =
+            GetNode<CustomCheckBox>(BiodiversityNearbyPatchIsFreePopulationPath);
+        lowBiodiversityLimitSpinBox = GetNode<SpinBox>(LowBiodiversityLimitPath);
+        maximumSpeciesInPatchSpinBox = GetNode<SpinBox>(MaximumSpeciesInPatchPath);
+        moveAttemptsPerSpeciesSpinBox = GetNode<SpinBox>(MoveAttemptsPerSpeciesPath);
+        mutationsPerSpeciesSpinBox = GetNode<SpinBox>(MutationsPerSpeciesPath);
+        newBiodiversityIncreasingSpeciesPopulationSpinBox =
+            GetNode<SpinBox>(NewBiodiversityIncreasingSpeciesPopulationPath);
+        protectMigrationsFromSpeciesCapCheckBox = GetNode<CustomCheckBox>(ProtectMigrationsFromSpeciesCapPath);
+        protectNewCellsFromSpeciesCapCheckBox = GetNode<CustomCheckBox>(ProtectNewCellsFromSpeciesCapPath);
+        refundMigrationsInExtinctionsCheckBox = GetNode<CustomCheckBox>(RefundMigrationsInExtinctionsPath);
+        strictNicheCompetitionCheckBox = GetNode<CustomCheckBox>(StrictNicheCompetitionPath);
+        speciesSplitByMutationThresholdPopulationAmountSpinBox =
+            GetNode<SpinBox>(SpeciesSplitByMutationThresholdPopulationAmountPath);
+        speciesSplitByMutationThresholdPopulationFractionSpinBox =
+            GetNode<SpinBox>(SpeciesSplitByMutationThresholdPopulationFractionPath);
+        useBiodiversityForceSplitCheckBox = GetNode<CustomCheckBox>(UseBiodiversityForceSplitPath);
+
+        currentGenerationLabel = GetNode<Label>(CurrentGenerationLabelPath);
+        runStatusLabel = GetNode<Label>(RunStatusLabelPath);
+        finishOneGenerationButton = GetNode<Button>(RunGenerationButtonPath);
+        runOneStepButton = GetNode<Button>(RunStepButtonPath);
+        abortButton = GetNode<Button>(AbortButtonPath);
+        playWithCurrentSettingButton = GetNode<Button>(PlayWithCurrentSettingPath);
+
+        autoEvoResultsLabel = GetNode<CustomRichTextLabel>(ResultsLabelPath);
+        historyListMenu = GetNode<CustomDropDown>(HistoryListMenuPath);
+
+        speciesPreview = GetNode<SpeciesPreview>(SpeciesPreviewPath);
+        hexPreview = GetNode<CellHexPreview>(HexPreviewPath);
+        speciesListMenu = GetNode<CustomDropDown>(SpeciesListMenuPath);
+        speciesDetailsLabel = GetNode<CustomRichTextLabel>(SpeciesDetailsLabelPath);
+        evolutionaryTree = GetNode<EvolutionaryTree>(EvolutionaryTreePath);
+
+        exitConfirmationDialog = GetNode<CustomConfirmationDialog>(ExitConfirmationDialogPath);
+
+        // Init game
+        gameProperties = GameProperties.StartNewMicrobeGame(new WorldGenerationSettings());
```

Comment:
```suggestion
        gameProperties = GameProperties.StartNewMicrobeGame(new WorldGenerationSettings(), true);
```
That true should be passed here so that the game is in freebuild mode, if the player decides to play in the world.

Reply:
Can't do this, freebuild generates random species that make the tree a real mess.

Reply:
Okay, then, you need to transform the game into a freebuild state later either before entering the game as the player or just after this line. Might need to add a new method if there isn't one already, and that should be commented that that method specifically won't create the freebuild starting species.

Reply:
I see you did this already in a later commit.